staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4233,6 +4233,12 @@ func (d *ConfigMapDescriber) Describe(namespace, name string, describerSettings
 			w.Write(LEVEL_0, "%s:\n----\n", k)
 			w.Write(LEVEL_0, "%s\n", string(v))
 		}
+		w.Write(LEVEL_0, "\nBinaryData\n====\n")
+		for k, v := range configMap.BinaryData {
+			w.Write(LEVEL_0, "%s: %s bytes\n", k, strconv.Itoa(len(v)))
```

Comment:
Please confirm that using `len` here is the right way.

I know [`encoding/binary#Size`](https://pkg.go.dev/encoding/binary#Size) exists but I know nothing, Jon Snow.

Reply:
Found this, https://stackoverflow.com/questions/31733145/size-of-a-byte-array-golang/31736829#comment51402266_31733145. I think they both will return the same because `data_type_size` will be 1 since `byte` type will be size 1
https://play.golang.org/p/4WCqUlvLoq2

Reply:
Thanks for taking a look!